### PR TITLE
Anthony Guevara - [UIUX/Challenge 1 - Karaoke needs words]

### PR DESCRIPTION
**Pull request by zuphu**
UIUX/Challenge 1 - Karaoke needs words

### CODE REVIEW

File: uiux/challenge-1/src/App.js [hunk 1]
```diff
@@ -0,0 +1,217 @@
...
+    var fullScreen = {
+      height: '100vh',
+      background: 'linear-gradient(to right, #1a2a6c, #b21f1f, #fdbb2d)'
+    }
+
+    var font = {
+      color:'orange'
+    }
+
+    return (
```
Comment (musabmasood) [inline uiux/challenge-1/src/App.js]: App.js should be importing components and render them. Why is rendering being done directly inside App.js?

File: uiux/challenge-1/src/App.js [hunk 2]
```diff
@@ -0,0 +1,217 @@
...
+  constructor() {
+    super();
+    this.keyDownFunction = this.keyDownFunction.bind(this);
+    this.handleClick = this.handleClick.bind(this);
+    this.handleArtistNameChange = this.handleArtistNameChange.bind(this);
+    this.searchForArtist = this.searchForArtist.bind(this);
+    this.searchForTracks = this.searchForTracks.bind(this);
+
+    // initialize states
+    this.state = {
```
Comment (musabmasood) [inline uiux/challenge-1/src/App.js]: Why is there no use of Redux for global state management of the application?

**Comment by musabmasood** [comment]
Why is there no separation of smart and dumb components for the application?

**Comment by zuphu** [comment]
Updated to address requested changes
@musabmasood

**Pull request closed without merging**